Core/OfficeDevPnP.Core/Utilities/Themes/ThemeUtility.cs

```diff
@@ -48,6 +48,36 @@ public static String GetThemeAsJSON(String primaryColor, String bodyTextColor, S
 
             return json;
         }
+
+        /// <summary>
+        /// Return an out of the box modern theme as a JSON string
+        /// </summary>
+        /// <param name="theme">Name of the out of the box theme</param>
+        /// <returns></returns>
+        public static String GetOOBModernThemeJson(OOBTheme theme)
+        {
+            switch (theme)
```

Comment:
Interesting. I just had a look at the "Change the Look" options and the theme colors. They seem to be embedded in the JS bundle. So no API to get the colors from (which I expected). But how do we detect if the default colors change? (Does it even matter?)

Reply:
True. Even I did not see any API which provides the OOB theme information. So one option is to use the code in the PR and as soon as there is one I will update the code.